### PR TITLE
fix(onboarding): cleaned up/imporved images

### LINKED ISSUE #5829: onboarding images improvements
- [x] Status logo is smaller than in figma, needs to be resized
- [x] Logo looks a bit pixelated
- [x] Some images through the onboarding needs their size adjusted to match figma (some are too small)

### PR DESCRIPTION
**Pull request by alexandraB99**
Closes #5829

### What does the PR do
- Status logo changed to 127.88x127.88
- logo pixelated fixed
- existing images adjusted, also cleaned up onboarding images folder

### Affected areas
onboarding

### CODE REVIEW

**Review by alaibe — APPROVED** [review]

File: ui/app/AppLayouts/Chat/panels/EmptyChatPanel.qml [hunk 1]
```diff
@@ -20,7 +20,7 @@ Item {
         id: walkieTalkieImage
         anchors.horizontalCenter: parent.horizontalCenter
         anchors.verticalCenter: parent.verticalCenter
-        source: Style.png("onboarding/chat@2x")
+        source: Style.png("chat/chat@2x")
```
Comment (endulab) [inline ui/app/AppLayouts/Chat/panels/EmptyChatPanel.qml:23]: @alexandraB99 I am thinking about using 2x and 3x. According to docs, on Mac, you need to use only "chat/chat" and the system will automatically choose 2x or 3x. Here you explicitly choose 2x. Do you have any thoughts ?

File: ui/app/mainui/SplashScreen.qml [hunk 2]
```diff
@@ -19,9 +19,10 @@ Item {
 
     AnimatedImage {
         id: splashLogo
+        width: 127.88
+        height: 127.88
         anchors.centerIn: parent
-        scale: 0.5
-        source: Style.gif("status_splash")
+        source: Style.gif("status_splash_"+(Style.current.background === "#ffffff" ? "light" : "dark"))
```
Comment (endulab) [inline ui/app/mainui/SplashScreen.qml:25]: You can use Style.current.name ( "light" | "dark" ).

File: ui/app/mainui/SplashScreen.qml [hunk 3]
```diff
@@ -19,9 +19,10 @@ Item {
 
     AnimatedImage {
         id: splashLogo
+        width: 127.88
```
Comment (endulab) [inline ui/app/mainui/SplashScreen.qml:22]: why .88 ?

**Review by endulab — CHANGES_REQUESTED** [review]

File: ui/app/AppLayouts/Chat/panels/EmptyChatPanel.qml (hunk 1 above)
Comment (alexandraB99) [reply ui/app/AppLayouts/Chat/panels/EmptyChatPanel.qml:23]: @endulab I don't. this was there from before, I just changed the path to it :) Fingerprint is also used like this `source: Style.png("onboarding/fingerprint@2x")` in TouchIDAuthView, maybe @borismelnik knows something?

File: ui/app/mainui/SplashScreen.qml (hunk 3 above)
Comment (alexandraB99) [reply ui/app/mainui/SplashScreen.qml:22]: FIG

File: ui/app/mainui/SplashScreen.qml (hunk 2 above)
Comment (alexandraB99) [reply ui/app/mainui/SplashScreen.qml]: done

File: ui/app/mainui/SplashScreen.qml (hunk 3 above)
Comment (endulab) [reply ui/app/mainui/SplashScreen.qml:22]: 🤦 right

**Review by endulab — APPROVED** [review]

**Pull request merged**